views/letter_value_partial.erb

```diff
@@ -0,0 +1,4 @@
+  <p> <% k.to_s.each_char do |char|%></p>
+    <% letter_value = Scrabble::ScoreWord.score_letter(char)%>
```

Comment:
There is a preference for not calling these methods directly in your views, so it would be more clean if this came from your scrabble-site.rb file